util/downloadUrlByOS.ts

```diff
@@ -0,0 +1,28 @@
+import type { UserOS } from '../types/userOS';
+
+export const downloadUrlByOS = ({
+  userAgent,
+  userOS,
+  version,
+  bitness,
+}: {
+  userAgent?: string | undefined;
```

Comment:
Can this type be extracted to a `type DownloadUrlByOS =` on this file? Just to keep the function clean 🙈

Reply:
ahah I was just asking the question bellow :p

Reply:
🙈